src/applications/vaos/containers/CommunityCarePreferencesPage.jsx

```diff
@@ -127,6 +132,16 @@ const uiSchema = {
     phone: {
       ...phoneUI(),
       'ui:required': data => data.hasCommunityCareProvider,
+      'ui:help': (
+        <AlertBox
+          status="info"
+          headline="We’ll try to schedule your appointment with your preferred community provider"
+        >
+          If we aren’t able to schedule this appointment with your preferred
+          provider, we’ll make every effort to schedule your appointment with
+          another community provider closest to your home.
+        </AlertBox>
+      ),
```

Comment:
Hi @jbalboni , the layout for the alert message is to display at the end of communityCareProvider uiSchema.  Visually it is under the phone input text box, but inserting the alertbox within the phone uiSchema describes the wrong property. How can i have it display in the desired location and describe the communityCareProvider object?

Reply:
Typically what we do in this situation is create an empty field, like this: https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/forms/common-patterns-for-building-forms#creating-a-block-of-text-with-no-associated-fields

In that example, you would put your component in place of the text used for `ui:description`.

Reply:
Thanks, worked out perfectly